Lib/test/test_c_locale_coercion.py

```diff
@@ -143,12 +144,15 @@ def get_child_details(cls, env_vars):
 
 
 # Details of the shared library warning emitted at runtime
-LEGACY_LOCALE_WARNING = (
+if test.support.is_android:
```

Comment:
Let's make the check here:

         if C_LOCALE_STREAM_ENCODING == "ascii":

That way, if any other platforms start using UTF-8 by default in the C locale, we'll just need to change the test logic in one place.

Reply:
Oops, had a typo in my suggested check (I wrote 'ascii' when I meant 'utf-8'). Fixed now :)

Reply:
Ok